lib/yao/resources/role.rb

```diff
@@ -13,6 +13,32 @@ def get_by_name(name)
         self.list.find {|role| role.name == name }
       end
       alias find_by_name get_by_name
+
+      def grant(role_name, to:, on:)
+        role   = Yao::Role.get_by_name(role_name)
+        user   = Yao::User.get_by_name(to)
+        tenant = Yao::Tenant.get_by_name(on)
+
+        client.put path_for_grant_revoke(tenant, user, role) do |req|
+          req.headers['Content-Type'] = 'application/json'
+        end
```

Comment:
* shortcut method `GET/PUT/POST/DELETE` is available: https://github.com/Joe-noh/yao/blob/grant-and-revoke-role/lib/yao/resources/restfully_accessible.rb#L12
* Request header `'application/json` is automatically assigned. https://github.com/Joe-noh/yao/blob/grant-and-revoke-role/lib/yao/client.rb#L48 Maybe this block is not needed.